std_msgs/Header header
std_msgs/Int16[] ids
